package.json

```diff
@@ -52,8 +52,8 @@
 		"lerna": "^3.4.0",
 		"prettier-eslint-cli": "^4.7.0",
 		"raf": "^3.4.1",
-		"react": "^16.8.1",
-		"react-dom": "^16.8.1",
+		"react": "^16.9.0-alpha.0",
```

Comment:
I don't think we should be committing any alpha versions, lets try to only rely on stable versions.

Reply:
This version is the only one that supports putting asynchronous logic on `act(` of react-testing-library.

Reply:
https://github.com/facebook/react/releases/tag/v16.9.0-alpha.0

Reply:
Since this is only for a test case, can we just disable to test case until a stable version is released?

Reply:
Makes sense. However our coverage will be decreased until the next React release.